tm-notify/src/main.rs

```diff
@@ -187,9 +194,19 @@ async fn notify(game_id: &str, message: String, webhook: &str) -> Result<()> {
     }
 }
 
+fn final_scoring_message(game: &ViewGameResponse) -> String {
+    let mut factions: Vec<_> = (&game.factions.as_ref().unwrap()).iter().collect();
```

Comment:
Something like this? (Won't compile)

```suggestion
fn final_scoring_message(game: &ViewGameResponse) -> Option<String> {
    let mut factions = match game.factions {
        Some(f) => f.clone(),
        None => return
     };
```

I know that this method is only invoked if the game is finished and therefore... but I like to avoid unwrap.